src/main/steps/connection-summary/content.ts

```diff
@@ -0,0 +1,39 @@
+import { TranslationFn } from '../../app/controller/GetController';
+
+const en = ({ partner }) => ({
+  title: 'You can use English or Welsh courts to get a divorce',
+  line1: 'Your answers indicate that you can apply for a divorce in England and Wales because:',
+  line2: `both you and your ${partner} are habitually resident`,
```

Comment:
Is it possible that this go into `you-can-use-english-welsh-courts/content.ts`, then you can reuse `enHabitualAndDomicileHelp`.